src/util/saveArticles.ts

```diff
@@ -131,7 +131,7 @@ async function getAllArticlesToKeep(articleDetailXId: RKVS<ArticleDetail>, dump:
       try {
         const mainPage = isMainPage(articleId)
         const renderer = mainPage ? mainPageRenderer : articlesRenderer
-        const leadSectionId = dump.nodet && !articleDetail.contentmodel ? config.filters.leadSectionId : ''
+        const leadSectionId = dump.nodet && !articleDetail.contentmodel && !articleDetail.missing ? config.filters.leadSectionId : ''
```

Comment:
Shouldn't we apply the same condition in `saveArticles`? I just realize it already misses the condition on `!articleDetail.contentmodel`.